common/lib/xmodule/xmodule/js/src/video/02_html5_video.js

```diff
@@ -16,11 +17,82 @@
+         * be inserted (as returned by jQuery(selector) function), or a
+         * selector string which will be used to select an element. This is a
+         * required parameter.
+         *
+         * @param config - An object whose properties will be used as
+         * configuration options for the HTML5 video player. This is an
+         * optional parameter. In the case if this parameter is missing, or
+         * some of the config object's properties are missing, defaults will be
+         * used. The available options (and their defaults) are as
+         * follows:
+         *
+         *     config = {
+         *
+         *        videoSources: [],   // An array with properties being video
+         *                            // sources. The property name is the
+         *                            // video format of the source. Supported
+         *                            // video formats are: 'mp4', 'webm', and
+         *                            // 'ogg'.
+         *
+         *          events: {         // Object's properties identify the
+         *                            // events that the API fires, and the
+         *                            // functions (event listeners) that the
+         *                            // API will call when those events occur.
+         *                            // If value is null, or property is not
+         *                            // specified, then no callback will be
+         *                            // called for that event.
+         *
+         *              onReady: null,
+         *              onStateChange: null
+         *          }
+         *     }
+         */
+        function Player(el, config) {
+            var errorMessage, lastSource, sourceList;
+
+            // Create HTML markup for individual sources of the HTML5 <video>
+            // element.
+            sourceList = $.map(config.videoSources, function(source) {
+                return [
+                    '<source ',
+                    'src="', source,
+            // Following hack allows to open the same video twice
+            // https://code.google.com/p/chromium/issues/detail?id=31014
+            // Check whether the url already has a '?' inside, and if so,
+            // use '&' instead of '?' to prevent breaking the url's integrity.
+                        (source.indexOf('?') === -1 ? '?' : '&'),
+                    (new Date()).getTime(), '" />'
+                ].join('');
+            });
+
+            // do common initialization independent of player type
+            this.init(el, config);
+
+            // Create HTML markup for the <video> element, populating it with
+            // sources from previous step. Set playback not supported error message.
+            errorMessage = [
+                gettext('This browser cannot play .mp4, .ogg, or .webm files.'),
+                gettext('Try using a different browser, such as Google Chrome.')
```

Comment:
@muhammad-ammar  if you were talking about this existing message then let's change this too with what sylvia suggested us.